dist/PathTools/lib/File/Spec/Win32.pm

```diff
@@ -249,7 +249,7 @@ Unlike just splitting the directories on the separator, leading empty and
 trailing directory entries can be returned, because these are significant
 on some OSs. So,
 
-    File::Spec->splitdir( "/a/b/c" );
+    File::Spec->splitdir( "/a/b//c" );
```

Comment:
That needs to be
```perl
File::Spec->splitdir( "/a/b//c/" );
```
(with a trailing slash) to match the output given.

PS: That whole method body could be replaced by `return split( m|[\\/]|, $directories, -1 );`

PPS: I'm not sure what the `\Z(?!\n)` dance is for. Compatibility with perl-5.004, which didn't have `\z` maybe?

Reply:
Oops, thank you and fixed.